ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitterSpec.scala

```diff
@@ -83,7 +106,6 @@ class ConfigCommitterSpec extends WordSpec with Matchers {
         case StepStop(actualLogEntry) =>
           actualLogEntry.hasRecordTime shouldBe true
           actualLogEntry.getRecordTime shouldBe buildTimestamp(theRecordTime)
-          actualLogEntry.hasConfigurationEntry shouldBe true
```

Comment:
Is that checked somewhere else?

Reply:
Yes, there's a test case below `produce a log entry ...` that does this check (and more) hence I removed this line to make the test cases independent.